spec/jobs/statistics_for_recent_comments_job_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe StatsForRecentCommentsJob do
+  describe '#perform' do
+    it 'calculates the statistics and sends a message' do
+      calculator = double(run: nil)
+      allow(StatsCalculator).to receive(:new).and_return(calculator)
+
+      StatsForRecentCommentsJob.new.perform
+
+      expect(StatsCalculator).to have_received(:new).with(channel: '#random')
```

Comment:
Maybe `StatsForRecentCommentsJob::DEFAULT_CHANNEL` instead of `'#random'`? Up to you, there's an argument for both ways.

Reply:
I agree with your proposed change, that's the way I usually do it. 9b06c3a